### DISCUSSION TITLE
ERROR: Could not find a version that satisfies the requirement syft (unavailable) ERROR: No matching distribution found for syft (unavailable)

### DISCUSSION THREAD

**Discussion by Spartan-119** [OpenMined/courses#295] [Course: Foundations of Private Computation]
I installed all the required libraries as mentioned in the requirements.txt.
Then I copy-pasted the command "pip install git+https://github.com/OpenMined/PySyft@dev#egg=syft"
I am getting the following error. Not sure what to make of this :/
![error](https://user-images.githubusercontent.com/36173893/112207319-fae26700-8c3c-11eb-870e-9ce5d52815ad.PNG)

**Comment by znreza** [comment]
Hi @Spartan-119 could you try installing like suggested [here](https://github.com/OpenMined/courses/discussions/288#discussioncomment-523630)?

`pip install syft-0.3.0.post0.dev754+g65a4aa120`

If it still doesn't work please post it in the #support channel in [OpenMined slack](https://openmined.slack.com/join/shared_invite/zt-nvhqkubp-haDCpXSasR3RbvJySIza8Q#/shared-invite/email). Someone from the dev team will be able to help you fix this!

**Reply by Spartan-119** [reply]
It's funny, I re-ran the earlier code again. This time it worked.
